scripts/es2k/setup_arm_infra.sh

```diff
@@ -41,7 +41,7 @@ function run_infrap4d () {
   export PATH=$P4CP_INSTALL/bin:$P4CP_INSTALL/sbin:$PATH
   export SDE_INSTALL=/opt/p4/p4sde
   export LD_LIBRARY_PATH=$P4CP_INSTALL/lib:$P4CP_INSTALL/lib64:$SDE_INSTALL/lib64:$SDE_INSTALL/lib:/usr/lib64:/usr/lib:/usr/local/lib64:/usr/local/lib
-  $P4CP_INSTALL/sbin/infrap4d
+  $P4CP_INSTALL/sbin/infrap4d &
```

Comment:
Is the `&` needed? From what I understand, with the `&`, the commands after it won't even execute. I feel like the previous one was correct?

Reply:
Ah I mistook it for `&&`. Nevermind